### DISCUSSION TITLE
Ingame Additional Ui-Button for Explorer Notes

### DISCUSSION THREAD

**Discussion by InfinityofphoenixTimo** [gameserverapp/gsa-mod-asa#3] [Suggestions]
A Additional button for the radial Menu, where the Player can check which Explorernotes he have. We could add here Pictures from the Spot of the Note and the Coordinates. That data can be extractet from the Playerfile via Gsa Maybe. The Ui could Show a List or a Quarter at this Notes where showed a Green Completed "✔" or a Red X, Also this could Addet via a Template to the GSA Profil dashboard for the Player Profiles.

**Comment by StevieSenpai** [comment]
in my opinion, I don't think it's necessary since the game has its own list with the explorer notes you have obtained, although it doesn't tell you the coordinates of the note, it does tell you how many you have/which ones you got
